src/AWS.Distro.OpenTelemetry.AutoInstrumentation/Plugin.cs

```diff
@@ -156,6 +164,20 @@ public void TracerProviderInitialized(TracerProvider tracerProvider)
                 tracerProvider.AddProcessor(new AwsBatchUnsampledSpanExportProcessor(exporter: unsampledSpanExporter, maxExportBatchSize: LambdaSpanExportBatchSize));
             }
         }
+
```

Comment:
I'm guessing for this to work, the customer will have to set the `OTEL_TRACES_EXPORTER=null` and set this new flag to true right? If so, do you mind updating the description with that info?

Reply:
Or maybe not. Just make it clear in the PR description how the sigv4 path is triggered and what is expected. Are we exporting the trace twice using the otlp exporter and the custom exporter?